dev/CalendarView/CalendarView.vcxitems

```diff
@@ -16,18 +16,18 @@
   </ItemGroup>
   <ItemGroup>
     <Page Include="$(MSBuildThisFileDirectory)CalendarView_themeresources.xaml">
-      <UseVisualStyle>Latest</UseVisualStyle>
+      <ControlsResourceVersion>Version2</ControlsResourceVersion>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
-    <Page Include="$(MSBuildThisFileDirectory)CalendarView_21h1_themeresources.xaml">
-      <UseVisualStyle>Latest</UseVisualStyle>
-      <Version>21H1</Version>
+    <Page Include="$(MSBuildThisFileDirectory)CalendarView_themeresources_v1.xaml">
+      <ControlsResourceVersion>Version1</ControlsResourceVersion>
+      <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
-    <Page Include="$(MSBuildThisFileDirectory)CalendarView_themeresources_v2.5.xaml">
```

Comment:
>CalendarView_themeresources_v2.5.xaml [](start = 46, length = 37)

I didn't remove this, just renamed it to CalendarView_themeresources_v1.xaml.
It's moved up to 23 line when merge and resolve conflict